requirements.txt

```diff
@@ -1,22 +1,23 @@
 cryptography>=3.4.8,<4
 feather-format>=0.4.1,<1
 jupyter>=1.0.0,<2
+importlib-metadata>=4.11.4,<5.0
 ipympl==0.7.0
 jupyter_contrib_nbextensions>=0.5.1,<1
-jupyterlab>=3.1.5,<4
+jupyterlab>=3.4.3,<4
 jedi>=0.17.2,<0.18
 matplotlib>=2.2.2,<3
-numpy>=1.16.3,<2
+numpy>=1.21.6,<2
 palettable>=3.3.0,<4
-pandas>=0.23.3,<1
-requests>=2.25.1,<3
+pandas>=1.3.5,<2
+requests>=2.27.1,<3
 scikit-image>=0.14.3,<=0.16.2
-scikit-learn>=0.19.1,<1
-scipy>=1.1.0,<2
-seaborn>=0.10.1,<1
-git+https://github.com/calico/spatial_lda.git@primary
-statsmodels>=0.11.1,<1
-tables>=3.6.1,<4
-umap-learn>=0.4.6,<1
-xarray>=0.12.3,<1
-tqdm>=4.54.1,<5
+scikit-learn>=0.24.2,<1
+scipy>=1.7.3,<2
+seaborn>=0.11.2,<1
+spatial-lda>=0.1.3,<1
```

Comment:
Can you check to confirm that this release of spatial-lda isn't significantly behind the github? They were slow to make a release before, which is why we were using the github

Reply:
I confirmed it with Adam a few days ago. He stated that 0.1.3 good to use.